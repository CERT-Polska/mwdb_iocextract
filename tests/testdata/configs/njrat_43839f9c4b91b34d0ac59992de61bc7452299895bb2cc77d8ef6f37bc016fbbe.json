{"c2": ["server77.ddns.net:5556"], "drop_name": "server.exe", "type": "njrat"}