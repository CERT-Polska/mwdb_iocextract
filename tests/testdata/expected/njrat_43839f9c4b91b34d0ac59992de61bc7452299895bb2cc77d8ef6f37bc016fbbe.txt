NetLoc server77.ddns.net:5556
Drop server.exe
